### DISCUSSION TITLE
Kubernetes Views within the cost summary PowerBi report

### DISCUSSION THREAD

**Discussion by Jshechtman** [microsoft/finops-toolkit#572] [Ideas]
In the azure preview portal, there is a preview view for Kubernetes Namespace. Here we're able to see the cost by namespace, idle charges, computer/networking/storage costs etc. This view is extremely useful and I'd really like to be able to show this within my powerbi reports as well. In a scenario where a platform/infrastructure team manages the Kubernetes clusters this would really help with chargeback/showback for these costs.

**Comment by flanakin** [comment]
Absolutely! I'll move this over to be an issue to track the feature request. There are some challenges to getting this but the request makes perfect sense.
